05/Textarea.js

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
 
 class Textarea extends React.Component {
-    render() {
-        return <textarea></textarea>
-    }
+	state = {
+		resize: false,
+	};
+	changeTextarea = e => {
+		const newValue = e.target.value;
+		this.props.onWrite(newValue);
+	};
+	getSnapshotBeforeUpdate(prevProps, prevState) {
+		return { resize: true }; // Zwracamy obiekt z informacją o resize
+	}
```

Comment:
Jeśli te dane nigdy się nie zmieniają w trakcje działania programu to ten kod coś robi? Raczej nie...
Jeśli w tym miejscu zrobisz warunku, które masz w `componentDidUpdate()` i będziesz zwracał `true` lub `false` - to wtedy m to sens. Wtedy we wspomnianej metodzie sparwdzasz tylko `if(snapshot.resize)` i wykonujesz odpowiednie działanie ;)

Reply:
to prawda, teraz zrozumiałem o co chodzi. poprzedni zapis chcialem po prostu wyslac zeby w codereview wyszlo co i jak